extension/test/unit/lib/magento/customer/getDiff-test.js

```diff
@@ -0,0 +1,26 @@
+const assert = require('assert')
+const step = require('../../../../../lib/magento/customer/getDiff')
+
+describe('getDiff', () => {
+  let context
+  let input
+
+  beforeEach(() => {
+    context = {}
+    input = {}
+  })
+
+  it('Should return correct arrays for adding and deleting products', async () => {
```

Comment:
it would be nice to have some more tests, 
1) maybe one that has variables mixed up instead of in order. Ya never know!
2) one that has nothing to delete
3) one that has nothing to add